### PR TITLE
Release 22.02 to master

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: .github/workflows/core-cicd-tests.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ jobs:
     name: Curl Tests
     runs-on: ubuntu-latest
     env:
-      DOT_CICD_BRANCH: master
+      DOT_CICD_BRANCH: release-22.02
```
Comment (dsilvam) [inline .github/workflows/core-cicd-tests.yml:61]: discard

File: .github/workflows/pre-release-process.yml [hunk 2]
```diff
@@ -13,16 +13,20 @@ on:
...
         required: false
         default: 'true'
 jobs:
   pre-release-process:
-    name: Automation of pre-release process
+    name: Automated Code Freeze process
     runs-on: ubuntu-latest
     env:
-      DOT_CICD_BRANCH: master
+      DOT_CICD_BRANCH: release-22.02
```
Comment (dsilvam) [inline .github/workflows/pre-release-process.yml:29]: discard

File: .github/workflows/release-process.yml [hunk 3]
```diff
@@ -7,7 +7,7 @@ jobs:
     name: Release Process Automation
     runs-on: ubuntu-latest
     env:
-      DOT_CICD_BRANCH: master
+      DOT_CICD_BRANCH: release-22.02
```
Comment (dsilvam) [inline .github/workflows/release-process.yml:10]: discard

File: .gitmodules [hunk 4]
```diff
@@ -1,5 +1,5 @@
 [submodule "dotCMS/src/main/enterprise"]
 	path = dotCMS/src/main/enterprise
 	url = git@github.com:dotCMS/enterprise.git
-	branch = master
+	branch = release-22.02
```
Comment (dsilvam) [inline .gitmodules:4]: discard

File: dotCMS/gradle.properties [hunk 5]
```diff
@@ -1,6 +1,6 @@
-dotcmsReleaseVersion=master
-coreWebReleaseVersion=next
-webComponentsReleaseVersion=next
+dotcmsReleaseVersion=22.02
```
Comment (dsilvam) [inline dotCMS/gradle.properties:1]: discard these three

**Review by dsilvam — APPROVED** [review]

File: dotCMS/build.gradle [hunk 6]
```diff
@@ -158,9 +158,9 @@ dependencies {
     /**** And now the libs we pull in from internal company sources - libs stored in ./plugins, ./bin, ./libs, the starter site, etc. ****/
     compile fileTree("src/main/plugins/com.dotcms.config/build/jar").include('plugin-com.dotcms.config.jar')
 
-    starter group: 'com.dotcms', name: 'starter', version: 'empty_20211201', ext: 'zip'
+    // starter group: 'com.dotcms', name: 'starter', version: 'empty_20211201', ext: 'zip'
```
Comment (erickgonzalez) [inline dotCMS/build.gradle:161]: We ship with empty starter, these changes needs to be reverteed

**Review by erickgonzalez — CHANGES_REQUESTED** [review]

**Review by nollymar — APPROVED** [review]

**Review by erickgonzalez — APPROVED** [review]

**Pull request closed without merging**